tests/test_copy.py

```diff
@@ -346,3 +346,25 @@ def test_commit_hash(src_repo, tmp_path):
     copier.run_copy(str(src_repo), tmp_path)
     assert tmp_path.joinpath("tag").read_text() == "1.0"
     assert tmp_path.joinpath("commit").read_text() == commit
+
+
+def test_value_with_forward_slash(tmp_path_factory):
+    src, dst = map(tmp_path_factory.mktemp, ("src", "dst"))
+    build_file_tree(
+        {
+            src / "{{ filename.replace('.', sep) }}.txt"
```

Comment:
```suggestion
            src / "{{ filename.replace('.', _copier_conf.sep) }}.txt"
```

Reply:
I mean to send the separator as a separate PR, after getting some feedback in #692 
But this works too